src/Updater.php

```diff
@@ -171,6 +183,45 @@ protected function executeGlobalActions(array $global_actions) {
     }
   }
 
+  /**
+   * Check if modules are enabled and installed.
+   *
+   * @param string $module
+   *   Module name where update definition is saved.
+   * @param string $update_definition_name
+   *   Update definition name. Usually same name as update hook.
+   *
+   * @return array
+   *   Returns array needed modules.
+   */
+  public function checkExpectedModules($module, $update_definition_name) {
+    $update_definitions = $this->configHandler->loadUpdate($module, $update_definition_name);
+    if (isset($update_definitions[UpdateDefinitionInterface::GLOBAL_ACTIONS])) {
+      if(isset($update_definitions[UpdateDefinitionInterface::GLOBAL_ACTIONS][UpdateDefinitionInterface::GLOBAL_CONDITION_EXPECTED_MODULES])){
+        return $this->checkExpectedModulesArray($update_definitions[UpdateDefinitionInterface::GLOBAL_ACTIONS][UpdateDefinitionInterface::GLOBAL_CONDITION_EXPECTED_MODULES]);
+      }
+    }
+    return [];
+  }
+
+  /**
+   * Check if modules are enabled and installed.
+   *
+   * @param array $global_actions
+   *   Array with list of expected modules.
+   *
+   * @return array
+   *   Returns array needed modules.
+   */
+  public function checkExpectedModulesArray(array $expected_modules) {
```

Comment:
If we remove usage of `checkExpectedModules` in checklist module, we can also remove `checkExpectedModules` function. And then we could rename `checkExpectedModulesArray` into `checkExpectedModules`, what would make function name nicer.